src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -163,13 +168,17 @@ describe('<PillButton> component', () => {
             expect(getButtonByName('This pill is in use up arrow')).toHaveClass(
                 'PillButton__pill PillButton__pill--isOpen PillButton__pill--isActive'
             );
+            expect(screen.queryByText(upArrowLabel)).not.toBeInTheDocument();
+            expect(screen.queryByText(clearLabel)).not.toBeInTheDocument();
```

Comment:
s/a

Reply:
:+1:

Reply:
We can't use toBeVisible for null.
Error message:
**received value must be an HTMLElement or an SVGElement.
Received has value: null**